build/dependencies.props

```diff
@@ -17,7 +17,7 @@
     <MicrosoftAspNetCoreTestHostPackageVersion>2.1.0-preview2-30220</MicrosoftAspNetCoreTestHostPackageVersion>
     <MicrosoftAspNetCoreTestingPackageVersion>2.1.0-preview2-30220</MicrosoftAspNetCoreTestingPackageVersion>
     <MicrosoftAspNetCoreWebUtilitiesPackageVersion>2.1.0-preview2-30220</MicrosoftAspNetCoreWebUtilitiesPackageVersion>
-    <MicrosoftExtensionsBuffersSourcesPackageVersion>2.1.0-preview2-30220</MicrosoftExtensionsBuffersSourcesPackageVersion>
+    <MicrosoftExtensionsBuffersSourcesPackageVersion>2.1.0-preview2-pk-corefx0-16426</MicrosoftExtensionsBuffersSourcesPackageVersion>
```

Comment:
For my knowledge, why was pk-corefx0 added to the version?

Reply:
It's my feature branch build, so I can test against other repos that had to be changed for new corefx.


I'll run Upgrade Repos automation after merging to fix things up.